test/common/performance/OWNERS

```diff
@@ -0,0 +1,10 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
```

Comment:
Need this file in test/test_images/performance also

Reply:
done